lib/src/components/supa_email_auth.dart

```diff
@@ -40,45 +39,55 @@ class MetaDataField {
 
 /// UI component to create email and password signup/ signin form
 class SupaEmailAuth extends StatefulWidget {
-  /// Whether the user is sining in or signin up
-  final SupaAuthAction authAction;
+  /// The URL to redirect the user to when clicking on the link on the
+  /// confirmation link after signing up.
+  final String? redirectTo;
 
-  /// `redirectUrl` to be passed to the `.signIn()` or `signUp()` methods
+  /// Callback for the user to complete a sign in.
+  final void Function(AuthResponse response) onSignInComplete;
+
+  /// Callback for the user to complete a signUp.
   ///
-  /// Typically used to pass a DeepLink
-  final String? redirectUrl;
+  /// If email confirmation is turned on, the user is
```

Comment:
Maybe I'm reading this wrong, but I dont get this description. `The user is`, is there something missing here?

Reply:
This is very much a typo! Thanks for finding it, and I will fix it!